src/constants/theme.js

```diff
@@ -23,11 +23,15 @@ const overrides = {
 	cerulean: {},
 	cosmo: {},
 	cyborg: {
-		chatButtonBackground: colors.dark
+		chatButtonBackground: colors.dark,
+		mentionBackgroundColor: colors.info,
```

Comment:
need to add `mentionBackgroundColor` and `mentionForegroundColor` to the block below for light themes?